01_fib/01_fib.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+func fib(n int) int {
+	var num1 = 0
+	var num2 = 1
+	var sum = 0
+	fmt.Fprintln(out, num2)
```

Comment:
It's better to decouple fibonacci sequence calculation from IO. https://github.com/anz-bank/go-training/issues/37

Reply:
As discussed with @juliaogris, this issue will be addressed after next session.